layouts/soarian/r4/clinical/medications.html

```diff
@@ -0,0 +1,5 @@
+<p>Resources in the Medications category are used to support the immunization and medication processes.</p>
+
+<p>Click <a href="https://hl7.org/fhir/r4/medications-module.html">here</a> to see the detailed categorization and full set of resources that fall into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.</p>
```

Comment:
We are not supposed to use the "Click here" construction wit h links anymore. Please change this sentence to 

Select <link> for the detailed categorization and full set of resources that fill into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.

PRODOC-160857.

Reply:
<p><a href="https://hl7.org/fhir/r4/medications-module.html">select</a> for the detailed categorization and full set of resources that fall into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.</p>. Did you mean like this way where 'select' is clickable which redirects page to https://hl7.org/fhir/r4/medications-module.html or did you mean to say the whole sentence is clickable ?

Reply:
Apologies, I meant to say "Select https://hl7.org/fhir/r4/medications-module.html for the detailed categorization and full set of resources that fill into this category in the HL7® FHIR® Standard." Make https://hl7.org/fhir/r4/medications-module.html a link.
Thank you!